Packs/MailSenderNew/Integrations/MailSenderNew/MailSenderNew.py

```diff
@@ -234,6 +234,10 @@ def create_msg():
     """
     # Collect all parameters
     to = argToList(demisto.getArg('to'))
```

Comment:
you can omit this line, since you added the if/else and you have the same line in line 240